rollup.config.js

```diff
@@ -28,7 +28,7 @@ function config(plugins) {
 
         plugins: plugins,
         // Fremtind packages are marked as internal so that packages that depend on each other don't get inlined in each other
-        external: ["react", "nanoid", /nrk\/core/, "classnames", /@babel\/runtime/, ...allFremtindPackagesNames],
+        external: ["react", "nanoid", "classnames", /@babel\/runtime/, ...allFremtindPackagesNames],
```

Comment:
foreslår at du bare kommenterer ut, lager et issue for å skru på igjen når vi kan droppe patchene.